assets/terraform-modules/bootkube/assets.tf

```diff
@@ -31,26 +31,26 @@ resource "local_file" "bootstrap-scheduler" {
 resource "local_file" "kube-apiserver" {
```

Comment:
@invidian instead of this, can you please cherry-pick this commit https://github.com/kinvolk/lokomotive/pull/1155/commits/a92526168af4a3bfeef310074f0e17e265adb9e7?

Reply:
Sure :)